docs/source/topics/simpy.rst

```diff
@@ -79,32 +79,79 @@ to gradually convert simulations.
     The :py:mod:`usim.py` documentation also describes how compatibility objects
     can be used directly in native μSim activities.
 
+Interactions between μSim and SimPy
+-----------------------------------
+
+The :py:mod:`usim.py` compatibility layer allows to use SimPy elements in μSim
+and vice versa. This works by translating the fundamental elements of each framework:
+
+* a Simpy :py:class:`~usim.py.Event` can be ``await``\ ed in a μSim activity, and
+* a μSim :term:`activity` can be ``yield``\ ed by a SimPy Process.
+
+Both approaches *return* the value or *raise* any errors of their activity or event.
+This gives full access to all SimPy features from μSim --
+however, there is no equivalent to μSim's ``async for`` and ``async with`` in SimPy.
+
+.. code:: python3
+
+    >>> from usim import time
+    >>> def car(env):
+    ...     """Partially migrated SimPy process"""
+    ...     trip_duration = 2
+    ...     parking_duration = 5
+    ...     while True:
+    ...         print(f'Start parking at {env.now}')
+    ...         yield (time + parking_duration)
+    ...
+    ...         print(f'Start driving at {env.now}')
+    ...         yield (time + trip_duration)
```

Comment:
Do you mind adding the actual start of the simulation? For the usim example below you actually to the run.

Reply:
Added simulation start and output as for the others.